ui/tests/acceptance/pageobjects/helloPage.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  url: function () {
+    return this.api.launchUrl + '/#/hello'
+  },
+
+  commands: {
+    navigateAndWaitTillLoaded: async function () {
+      const url = this.url()
+      return this.navigate(url).waitForElementVisible('@accountInput')
+    },
+
+    inputName: async function (name) {
+      await this
+        .waitForElementVisible('@accountInput')
+        .clearValue("@accountInput")
+        .setValue("@accountInput", name)
+      return this.waitForElementVisible("@submitButton")
+        .click("@submitButton")
+    },
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=5709816)